fix(a11y): final fix for WCAG 1.1.1 image alt should describe image content accurately (#17)
scottrigby/basekamp-next: content/events/eating-public.md

title: Eating in Public
images:
  - src: 2651_74825484898_732739898_2328830_4808758_n.jpg
    alt: Concrete block tower structure with FREE sign in outdoor public space
  - src: 2651_74810604898_732739898_2328421_1591262_n.jpg

>     alt: Coconuts with drawn faces on grass next to FREE NUTS sign

Reviewer: ```suggestion
    alt: Potatoes with drawn faces on grass next to FREE NUTS sign
```

  - src: 2651_74810634898_732739898_2328426_4963516_n.jpg

>     alt: Wooden box painted with red text reading dollar sign zero FREE MONEY BOX

Reviewer: ```suggestion
    alt: Wooden box painted with red text reading zero dollars, free money box
```

  - src: 2651_74810599898_732739898_2328420_1343892_n.jpg

>     alt: Collection of toys and objects on grass with sign about unwanted gifts and free store solution

Reviewer: ```suggestion
    alt: Collection of toys on grass with sign reading Problem: unwanted christmas presents, Solution: free store
```

  - src: 2651_74810624898_732739898_2328425_2069607_n.jpg
    alt: Outdoor shelf display with FREE STORE sign containing various items and stuffed animals
  - src: 2651_74812089898_732739898_2328514_8949_n.jpg
    alt: Red and black gardening containers with seedlings arranged on grass
  - src: 2651_74813709898_732739898_2328561_125802_n.jpg
    alt: Cardboard box with FREE text and anti-capitalism project label on grass
  - src: 2651_74825489898_732739898_2328831_6609689_n.jpg
    alt: Papaya seedlings on grass with sign marking anti-capitalism project
  - src: 2651_74825504898_732739898_2328834_7399955_n.jpg

>     alt: Large crowd of people seated at tables dining indoors at community meal event with FEAST signage

Reviewer: ```suggestion
    alt: Empty cardboard box showing 'Free' spray painted in big letters, and 'An Anti-capitalism project' written below
```
